econml/iv/dr/_dr.py

```diff
@@ -689,37 +2133,41 @@ class LinearIntentToTreatDRIV(StatsModelsCateEstimatorMixin, IntentToTreatDRIV):
     """
 
     def __init__(self, *,
-                 model_Y_X,
-                 model_T_XZ,
+                 model_y_xw="auto",
+                 model_t_xwz="auto",
                  flexible_model_effect,
+                 z_propensity="auto",
                  featurizer=None,
                  fit_cate_intercept=True,
                  cov_clip=.1,
                  cv=3,
                  mc_iters=None,
                  mc_agg='mean',
+                 opt_reweighted=False,
                  categories='auto',
                  random_state=None):
-        super().__init__(model_Y_X=model_Y_X,
-                         model_T_XZ=model_T_XZ,
+        super().__init__(model_y_xw=model_y_xw,
+                         model_t_xwz=model_t_xwz,
                          flexible_model_effect=flexible_model_effect,
+                         model_final=None,
+                         z_propensity=z_propensity,
                          featurizer=featurizer,
                          fit_cate_intercept=fit_cate_intercept,
-                         model_final=None,
                          cov_clip=cov_clip,
                          cv=cv,
                          mc_iters=mc_iters,
                          mc_agg=mc_agg,
-                         opt_reweighted=False,
-                         categories=categories, random_state=random_state)
+                         opt_reweighted=opt_reweighted,
+                         categories=categories,
+                         random_state=random_state)
 
     def _gen_model_final(self):
         return StatsModelsLinearRegression(fit_intercept=False)
```

Comment:
If we switch design, we should should set: `fit_intercept=self.fit_cate_intercept`